source/MaterialXContrib/MaterialXMaya/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ if (EXISTS "${MAYA_DEBUG_DIR}" OR EXISTS "${MAYA_RELEASE_DIR}")
         SOVERSION ${MATERIALX_NODE_VERSION}
         PREFIX ""
         SUFFIX ".mll"
-        DEBUG_POSTFIX ${MATERIALX_DEBUG_POSTFIX}
+        DEBUG_POSTFIX ""
```

Comment:
The suffix was added in https://github.com/autodesk-forks/MaterialX/pull/627 Having the suffix complicates Maya plugin loading. Luckily, it is a leaf node in the build dependency graph so reverting this won't affect anyone else.